web/src/compose.js

```diff
@@ -89,8 +89,8 @@ export function update_video_chat_button_display() {
 export function compute_show_audio_chat_button() {
     const available_providers = page_params.realm_available_video_chat_providers;
     if (
-        available_providers.jitsi_meet &&
-        page_params.realm_video_chat_provider === available_providers.jitsi_meet.id
+        page_params.realm_video_chat_provider === available_providers.jitsi_meet.id &&
+        page_params.jitsi_server_url
```

Comment:
@sahil, I have added this commit. It is related to the changes made in #25922, which were merged a couple of days back. It was missing an edge case, and we make changes around here in the following commit. Therefore, I have included this commit in this PR. I don't know if removing `available_providers.jitsi_meet` was really necessary, but we also don't check it in `compute_show_video_chat_button()`.

Reply:
Ok, I also think that we do not need `available_providers.jitsi_meet` check.